docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
     command: python manage.py runserver_plus 0.0.0.0:8000
     depends_on:
      - db
+     - selenium
```

Comment:
@michaelglenister we do not have separate staging and prod docker files right now.
@jbothma Should we separate them out in this change or should we do that in a separate story?
Not sure how much change it will be